src/main/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/SyncController.kt

```diff
@@ -18,7 +17,6 @@ import uk.gov.justice.digital.hmpps.manageusersapi.service.SyncStatistics
 import uk.gov.justice.digital.hmpps.manageusersapi.service.UserSyncService
 
 @RestController
-@Validated
```

Comment:
Removed due to incompatibility https://github.com/spring-projects/spring-framework/issues/23499